packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useCodeComparisonStyles = makeStyles({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    '> *': {
+      width: '100%',
+    },
+
+    '> :not(:last-child)': {
+      marginRight: '5%',
+    },
+  },
+});
+
+export const CodeComparison = (props: { children: React.ReactElement[] }) => {
+  const { children } = props;
+  const classes = useCodeComparisonStyles();
+
+  return <div className={classes.root}>{children}</div>;
+};
+
+export const CodeLanguages: { [key: string]: string } = {
+  html: 'HTML',
+  css: 'CSS',
+  js: 'JavaScript',
+  jsx: 'React',
+};
+export const CodeExample = (props: { title?: string; children: React.ReactElement }) => {
+  const { title, children } = props;
```

Comment:
**suggestion:** quite a lot logic happening here, would you mind to add a test for it ?

Reply:
I didn't go through with this given the nature of the utils but I'll go ahead and create some tests. Do we have a preferred structure for tests in Storybook or should I just add to the same folder as the `utils` file?

Reply:
yeah I'd go just with `utils.test.tsx` side by side. Im afraid there is no react-testing-library in place for react-examples

Reply:
Would it make sense to add it for consistency or just stay with enzyme?

Reply:
RTT (react testing lib) is already used via sinlge source of truth policy, thus it should just work without need to add things. Although there might be some issues because RTT depends on version of react-dom which might not be supported in RTT ( we ran into some issues with @tringakrasniqi  AFAIR )